restapi/src/main/java/io/stargate/web/docsapi/service/util/DocsApiUtils.java

```diff
@@ -233,6 +233,17 @@ public static String convertJsonToBracketedPath(String path) {
     return newPath.toString();
   }
 
+  public static String[] bracketedPathAsArray(String bracketedPath) {
+    String[] split = bracketedPath.split("\\]\\[");
```

Comment:
can we have a compiled pattern for this split? I think this will not take fast path:

```
        /* fastpath if the regex is a
         (1)one-char String and this character is not one of the
            RegEx's meta characters ".$|()[{^?*+\\", or
         (2)two-char String and the first char is the backslash and
            the second is not the ascii digit or ascii letter.
         */
```